src/main/java/com/example/bookstore/controller/AdminController.java

```diff
@@ -0,0 +1,59 @@
+package com.example.bookstore.controller;
+
+import com.example.bookstore.entity.User;
+import com.example.bookstore.service.UsersService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/admin")
+public class AdminController {
```

Comment:
```suggestion
public class UsersController {
```
file này trong package controller.admin sẽ ok hơn nha cu